api/docker-compose.prod.yml

```
@@ -0,0 +1,5 @@
+services:
+  mysql:
+    volumes:
+      - mysql_data:/var/lib/mysql
+      - ./init-production.sql:/docker-entrypoint-initdb.d/init.sql:ro
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)